components/nginx/scripts/start.py

```diff
@@ -8,8 +8,25 @@
     join('components', 'utils.py'),
     join(dirname(__file__), 'utils.py'))
 import utils  # NOQA
+
 NGINX_SERVICE_NAME = 'nginx'
+ctx_properties = utils.ctx_factory.get(NGINX_SERVICE_NAME)
+
+
+def check_response(response):
+    """Check if the response looks like a correct REST service response.
+
+    We can get a 200, or a 401 in case auth is enabled. We don't expect a
+    502, though, as this would mean nginx isn't correctly proxying to
+    the REST service.
+    """
+    return response.code in {200, 401}
 
 
 ctx.logger.info('Starting Nginx Service...')
 utils.start_service(NGINX_SERVICE_NAME, append_prefix=False)
+utils.systemd.verify_alive(NGINX_SERVICE_NAME, append_prefix=False)
+
+nginx_url = '{0}://127.0.0.1/api/v2.1/blueprints'.format(
```

Comment:
Similar to how we hit the endpoint in restservice start script (to check that the restservice can access ES correctly), here we can try to access restservice via nginx, to check if it's proxying correctly? Do we want this check to be more strict, and do we want it at all?